src/views/expired-password/Footer.js

```diff
@@ -35,6 +37,10 @@ export default View.extend({
         .doTransaction(function(transaction) {
           return transaction.cancel();
         })
+        .then(function() {
+          const authClient = self.options.appState.settings.authClient;
```

Comment:
- probably can just be `self.settigs.authClient` or `self.settings.getAuthClient()`. (see line 45 below)
- ditto for another file